config/locales/en.yml

```diff
@@ -29,6 +29,7 @@ en:
         empty_file_warning: "%{file_name} is empty"
         file_too_large: "The size of the uploaded file %{file_name} exceeds the maximum of %{max_size} MB."
         exist: "The Folder is already exist"
```

Comment:
"The folder already exists"
"The folder does not exist"

Reply:
Also, please add translations for es, fr, and pt.

You can just use the string "UPDATE ME" and we can add them later but they should exist for all of them